src/locales/zh.json

```
@@ -88,6 +88,7 @@
       "new_dao_withdraw": "每日新增解锁",
       "circulation_ratio": "锁定量占当前流通量比率",
       "live_cell": "未花费 Cell",
+      "dead_cell": "已死亡 Cell",
```

This word is inelegant in Chinese, how about
```suggestion
      "dead_cell": "已消费 Cell",
```

> Each Cell can be consumed only once. A non-consumed cell is known as a live cell. A consumed cell is known as a dead cell. Once dead, the cell can no longer be used.

Ref: https://docs.nervos.org/docs/basics/concepts/cell-model/